spec/watirspec/elements/td_spec.rb

```diff
@@ -70,4 +70,12 @@
     end
   end
 
+  describe "#column_header" do
+    it "returns the corresponding column header" do
+      td = browser.td(text: '1 331').column_header
+      expect(td.text).to eq 'Income tax'
+    end
+
+  end
+
```

Comment:
I think how we are doing this one is that if there is an empty line after the `describe` line, then there is an empty line before the closing `end`. At least I've been trying to do that. Keeping up with style things is hard. :-D I should do another pass with rubocop at some point.